docs/one-click-integrations/github-integration.md

```diff
@@ -100,18 +103,34 @@ You can also choose not to detect deployment automatically via GitHub and send y
 
 ## Automatic incident detection strategies {: id="incident-detection-strategy"}
 
-The Pulse GitHub integration detects incidents automatically using [pull request reverts](#gh-incident-pr-revert). You can also choose [not to detect incidents via GitHub](#gh-incident-not-detect).
+The Pulse GitHub integration can detect **incidents** automatically using the following strategies:
+
+-   [Pull request reverts](#gh-incident-pr-revert)
+-   [Pull request naming convention](#gh-incident-naming)
+
+You can also choose [not to detect incidents via GitHub](#gh-incident-not-detect).
 
 !!! note
-    Pulse incident detection mechanism bases on pull request. Thus, Pulse can only detect incidents automatically if you configured the deployment detection strategy to use [merged pull requests](#gh-deploy-merged-pr).
+    Pulse uses pull requests to detect incidents. Thus, Pulse can only detect incidents automatically from GitHub if you configure the deployment detection strategy to use [merged pull requests](#gh-deploy-merged-pr).
 
 ### Use pull request reverts (based on default branch) {: id="gh-incident-pr-revert"}
 
 -   Pulse bases incident detection on [pull request reverts](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/reverting-a-pull-request).
+
 -   Pulse considers an incident any pull request that **targets the default branch** of the repository merged from a branch whose name starts with `revert-`, getting the number of the reverted pull request from the branch name, `revert-<pull request number>`. If you change the name of the branch created by GitHub when you revert a pull request, Pulse may not be able to obtain the incident data correctly.
+
 -   The incident creation date is the timestamp when the reverted pull request was initially merged. If Pulse can't get the reverted pull request number from the branch name, the incident creation date is the timestamp of the first commit to the incident pull request.
+
 -   Pulse associates incidents to the system matching the repository name.
 
+### Use pull request naming convention (based on default branch) {: id="gh-incident-naming"}
+
+-   Pulse applies the same mechanism to detect incidents used for the [pull request reverts](#gh-incident-pr-revert) strategy. However, instead of considering branches whose name starts with `revert-`, Pulse uses the prefix that you define for the **branch name** or the **pull request title** to match the naming convention of your workflow.
+
+    ![Pulse custom incident detection](images/github-incident-naming-convention.png)
```

Comment:
clean: We could omit this screenshot here and update the main screenshot `github-incident-strategy.png` to have the naming convention option selected and showing the available configurations.

Reply:
See https://github.com/codacy/pulse-user-docs/pull/198#discussion_r1117385468. :heavy_check_mark: